samples/certs/Makefile

```diff
@@ -23,6 +24,12 @@ CITADEL_CN ?= Intermediate CA
 CITADEL_SAN_DNS ?= localhost
 # Additional variables are defined in %/intermediate.conf target below.
 
+#------------------------------------------------------------------------
+# variables: workload certs: eg VM
+WORKLOAD_DAYS ?= 365
+NAMESPACE ?= vm
```

Comment:
How about 'productpage' or 'test' - or wherever we're adding the example VM.
Not default.

Reply:
the vm is added to the `vm`1 namespace. Happy to put it elsewhere. That is where the docs put the sample VM integration today.

Reply:
If that's what the current doc does - we can keep it.